src/pratt2.rs

```diff
@@ -111,11 +111,33 @@ where
     fn associativity(&self) -> Associativity {
         self.associativity
     }
-    fn fold_infix(&self, lhs: O, op: Self::Op, rhs: O) -> O {
+    fn fold_infix(&self, lhs: O, op: Self::Op, rhs: O, _span: I::Span) -> O {
         (self.f)(lhs, op, rhs)
     }
 }
 
+impl<'a, I, O, E, A, F, Op> Operator<'a, I, O, E> for Infix<A, F, Op, (O, Op, O, I::Span)>
+where
+    I: Input<'a>,
+    E: ParserExtra<'a, I>,
+    A: Parser<'a, I, Op, E>,
+    F: Fn(O, Op, O, I::Span) -> O,
```

Comment:
No specific reason, no. Is there any particular way you think it could be made more explicit? Throughout chumsky I mostly use `O/U/V` for output types, `A/B/C` for parser types, `I` for input types, `E` for the `ParserExtra` parameter, and `F/G/H` for functions.

Reply:
For what it's worth, the I, O, E names I'm familiar with. One sees those everywhere in Chumsky.

A, B, C, and F, G, H aren't as obvious.